plugins/account_history_plugin/account_history_plugin.cpp

```diff
@@ -443,7 +443,7 @@ fc::variant account_history_plugin_impl::transaction_to_variant(const packed_tra
    };
 
    fc::variant pretty_output;
-   abi_serializer::to_variant(ptrx, pretty_output, resolver);
+   abi_serializer::to_variant(ptrx, pretty_output, resolver, chain_plug->chain().get_abi_serializer_max_time_ms());
```

Comment:
get_abi_serializer_max_time_ms of chain_controller is already removed. But I guess, we won't need to use account_history_plugin anymore since we have history_plugin now.